origin.spec

```diff
@@ -187,10 +188,9 @@ pushd _thirdpartyhacks
 popd
 export GOPATH=$(pwd)/_build:$(pwd)/_thirdpartyhacks:%{buildroot}%{gopath}:%{gopath}
 # Build all linux components we care about
-for cmd in oc openshift dockerregistry
-do
-        go install -ldflags "%{ldflags}" %{import_path}/cmd/${cmd}
-done
+go install -ldflags "%{ldflags}" %{import_path}/cmd/dockerregistry
+go install -ldflags "%{ldflags}" -tags=gssapi %{import_path}/cmd/openshift
+go install -ldflags "%{ldflags}" -tags=gssapi %{import_path}/cmd/oc
 go test -c -o _build/bin/extended.test -ldflags "%{ldflags}" %{import_path}/test/extended
```

Comment:
not yet, because we don't have a kerberos ticket server running. @enj is writing extended tests that will spin up a ticket server and exercise the kerberos auth